README.md

```diff
@@ -1 +1,1369 @@
-Example ForgeRock Platform Scripts
+
+# <a id="top"></a>Dynamic Scripting in ForgeRock Platform
+
+Three of ForgeRock Identity Platform products, [Access Management](https://www.forgerock.com/platform/access-management) (AM), [Identity Management](https://www.forgerock.com/platform/identity-management) (IDM), and [Identity Gateway](https://www.forgerock.com/platform/identity-gateway) (IG), allow to extend their functionality with dynamically applied, context-specific scripts written in JavaScript and Groovy and evaluated during the run time.
+
+Scripting is broadly used in the products and broadly covered across [ForgeRock Product Documentation](https://backstage.forgerock.com/docs/). There are many articles describing scripting environment and application, often in a context of particular task and supplied with examples. Other places in the documentation cover functionality that is not directly related to scripting, but can be employed by scripts.
+
+This writing aims at a quick introduction to scripting across the Platform—via comparing the scripting environments, references to the official documentation, and an example of scripting in each of the three products.
+
+## Contents
+
+* [Conclusion](#conclusion)
+* [Summary](#summary)
+* [A Simple Example of Scripting in ForgeRock Components](#example)
+    * [AM](#example-am)
+    * [IDM](#example-idm)
+    * [IG](#example-ig)
+* [References](#references)
+
+## <a id="conclusion"></a>Conclusion
```

Comment:
Conclusion in the beginning?

Reply:
I think it will depend on the main content, when it is finalized.

Reply:
It maybe named "Abstract" if it stays at the top.